pipelines.py

```diff
@@ -1,12 +1,19 @@
 from functools import partial
 
 import loaders
-from models import PyTorchUNet
-from postprocessing import BuildingLabeler, Resizer, CategoryMapper, MulticlassLabeler
 from steps.base import Step, Dummy
 from steps.preprocessing.misc import XYSplit
 from utils import squeeze_inputs
 
+from models import PyTorchUNet
+from postprocessing import BuildingLabeler, Resizer, CategoryMapper, MulticlassLabeler
+
+# Streaming version, only for single output model
+
+# from models import PyTorchUNetStream as PyTorchUNet
```

Comment:
@taraspiotr I think it is better to have a variable `stream_mode` in neptune.yaml and then in unet pipeline have if config.stream_mode: PytorchUnetStream else PyTorchUNet and the same for the postprocessing classes. If we really have to deal with this in imports (at its ok) having a global RUN_AS_STREAM=True and than conditional imports is better than commented imports